package/Dockerfile.shipyard-dapper-base

```
@@ -69,6 +70,7 @@ RUN LINT_VERSION=$(awk '/golangci-lint/ { print $2 }' /tools.mod) && \
     BUILDX_VERSION=$(awk '/docker.buildx/ { print $2 }' /tools.mod) && \
     curl -L "https://github.com/docker/buildx/releases/download/${BUILDX_VERSION}/buildx-${BUILDX_VERSION}.linux-${ARCH}" -o /usr/local/libexec/docker/cli-plugins/docker-buildx && \
     chmod 755 /usr/local/libexec/docker/cli-plugins/docker-buildx && \
+    curl -L https://raw.githubusercontent.com/jonmosco/kube-ps1/master/kube-ps1.sh -o /root/.local/bin/kube-ps1.sh && \
```

Since this ends up injecting code that runs automatically on `make shell`, it should track specific hashes:

```
```suggestion
    curl -L https://raw.githubusercontent.com/jonmosco/kube-ps1/1b8fe913b25ba857b84a94c3b1dbf7bb34f7caef/kube-ps1.sh -o /root/.local/bin/kube-ps1.sh && \
```